components/oku-circ.vue

```diff
@@ -78,6 +78,18 @@ export default {
 $light-blue: #3C96D2;
 $red: #D0011B;
 
+// Width scale factor for Olin
+$olin-width: 0.053vw;
```

Comment:
- 2 letters works.
- _re:  I'm seeing some overlap and tightness when testing it here_. Let me check.
- Notice the vertical alignment of the count within the circles is not vertical aligned to the center.